documentation/book/proc-changing-a-topic.adoc

```diff
@@ -62,3 +62,9 @@ endif::Kubernetes[]
 ----
 oc replace -f _<your-file>_
 ----
+
+.Additional resources
+* For more information about the API for `KafkaTopics`, see xref:type-KafkaTopic[`KafkaTopic` API documentation]
```

Comment:
TBH I would not understand what _API_ means. Is it the Resource description? Is it Topic Operator configuration?

Reply:
How about "`KafkaTopic` schema reference"?

Reply:
That sounds great ... I would know what to expect there.